pkgs/servers/urserver/default.nix

```diff
@@ -9,11 +9,11 @@
 
 stdenv.mkDerivation rec {
   pname = "urserver";
-  version = "3.6.0.745";
+  version = "3.9.0.2465";
 
   src = fetchurl {
-    url = "https://www.unifiedremote.com/static/builds/server/linux-x64/745/urserver-${version}.tar.gz";
-    sha256 = "1ib9317bg9n4knwnlbrn1wfkyrjalj8js3a6h7zlcl8h8xc0szc8";
+    url = "https://www.unifiedremote.com/static/builds/server/linux-x64/2465/urserver-${version}.tar.gz";
```

Comment:
Can we take the patch release number from version and put it in here?

Reply:
I was wondering about that myself, but I wasn't sure how to write that nicely. Would you suggest using a `let` binding, or can this be included directly in the string interpolation?

Reply:
Made an attempt, feel free to correct me.